learning/tour-of-beam/frontend/lib/pages/tour/screen.dart

```diff
@@ -161,37 +164,37 @@ class _Units extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Column(
-      children: const [
-        _Unit(),
-        _Unit(),
-      ],
+      children: ['ParDo one-to-one', 'ParDo one-to-many']
+          .map((e) => _Unit(unit: e))
+          .toList(),
     );
   }
 }
 
 class _Unit extends StatelessWidget {
-  const _Unit();
+  final String unit;
```

Comment:
`unit.unit` is vague. Let's name this `title`.

Reply:
I expect the `unit` type to change from `String` to `UnitModel` later. I've renamed it to `title` and will change it later if necessary.